src/pm-conversations/PmConversationList.js

```diff
@@ -25,45 +27,48 @@ type Props = $ReadOnly<{|
 /**
  * A list describing all PM conversations.
  * */
-export default class PmConversationList extends PureComponent<Props> {
-  handleUserNarrow = (user: UserOrBot) => {
-    this.props.dispatch(doNarrow(pm1to1NarrowFromUser(user)));
+export default function PmConversationList(props: Props) {
+  const handleUserNarrow = (user: UserOrBot) => {
+    props.dispatch(doNarrow(pm1to1NarrowFromUser(user)));
   };
 
-  handleGroupNarrow = (users: PmKeyUsers) => {
-    this.props.dispatch(doNarrow(pmNarrowFromUsers(users)));
+  const handleGroupNarrow = (users: PmKeyUsers) => {
+    props.dispatch(doNarrow(pmNarrowFromUsers(users)));
```

Comment:
Let's add a quick refactoring commit to have this component start using `useDispatch`, now that it's a functional component using Hooks. That will simplify the callers somewhat, as well as make this component more idiomatic again. As is, it winds up feeling sort of half-converted.

Reply:
Done.